src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,119 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
+
+
+## Step 1: Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry data](self-hosted/helm-configuration.mdx#telemetry) to help us understand usage patterns and prioritize improvements
+In air-gapped environments, you can disable telemetry by adding the following to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+Disabling telemetry will automatically stop Okteto CLI analytics for all developers.
```

Comment:
Im not sure but I think that we should disable it from the CLI too right? 
From my testings I recall it was enabled by default. Im still testing other things before this but I have it in my notes

Reply:
CLI is disabled automatically if you disable telemetry on the chart. Have you seen a different behavior?

Reply:
I can't tell for sure cause I already disable it. But I think I installed Okteto with telemtry disabled and then I had to disable it in the CLI. 

I can test it if you want from scratch, but we could add something like "Make sure that you have analytics disabled in the CLI" ?

Reply:
Try to enable the CLI analytics `okteto analytics enabled` and it should send metrics if it's disabled at the helm level. It would be a critical bug otherwise